lib/index.js

```diff
@@ -92,12 +94,48 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
         });
     }));
 
+  app.get(
+    adminPath,
+    ensureAuthenticated,
+    intParser(),
+    validate({query: 'account.pagination'}),
+    asyncHandler(async (req, res) => {
+      const {email, cursor = null, limit = 10} = req.query;
+      const {actor} = req.user || {};
+      const query = {'account.email': email};
+      if(cursor) {
+        query['account.id'] = {$gt: cursor};
+      }
+      const options = {limit, sort: {'account.id': -1}};
+      const records = await brAccount.getAll({actor, query, options});
+      res.json(records);
+    }));
```

Comment:
All of this should be inserted into the other `get` call where the TODO is:

https://github.com/digitalbazaar/bedrock-account-http/blob/enhancement/accountsByEmail-accountUpdate/lib/index.js#L88

There should not be a separate endpoint that bifurcates "admin" from other accounts -- that creates a false dichotomy that's an anti-pattern and disallows future more granular approaches where certain accounts can modify certain other subsets of accounts but not others, etc.

Reply:
woooo not sure on that one. the other get is optionallyAuthenticated and used in the registration process, that route is supposed to be insecure. this route can return pretty sensitive stuff and in fact needs a better project (right now I am getting everything including salt, password hard, etc.) Are you 100% absolutely sure you want a route which is intended for admins that should not even been accessible to the average user being open to optionalAuth?